db/migrate/20200906155253_add_visible_to_partners_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddVisibleToPartnersToItems < ActiveRecord::Migration[6.0]
+  def change
+    add_column :items, :visible_to_partners, :boolean, default: true
```

Comment:
```suggestion
    add_column :items, :visible_to_partners, :boolean, default: true, null: false
```
Could we add a database constraint to prevent this from being nil?

Reply:
Good point. Updated.